scripts/pre_push_hook.py

```diff
@@ -122,10 +146,10 @@ def _collect_files_being_pushed(ref_list, remote):
         ref_list: list of references to parse (provided by git in stdin)
         remote: the remote being pushed to
     Returns:
-        Tuple of lists of changed_files, files_to_lint
+        dict: Dict consisting of branch --> ists of changed_files, files_to_lint
```

Comment:
dict: Dict mapping branch names to 2-tuples of the form (list of changed files, list of files to lint)